lib/upload_validator_extensions.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module UploadValidatorExtensions
+  def validate(upload)
+    extension = File.extname(upload.original_filename)[1..-1] || ""
+
+    if extension == "encrypted"
+      original_filename = upload.original_filename.gsub(/.encrypted$/, "")
```

Comment:
The "`.`" in a regexp is "any character".

```suggestion
      original_filename = upload.original_filename.gsub(/\.encrypted$/, "")
```